templates/abandoned.html.twig

```diff
@@ -0,0 +1,3 @@
+GAME ABANDONNEE
```

Comment:
Missing translation

In twig it's as simple as `{{ "GAME ABANDONNEE"|trans }}`